docs/guides/migration_guide_v2.md

```diff
@@ -143,12 +192,14 @@ Tokens should be created in the console.
 
 The `scaleway_volume_attachment` was removed in version 2.
 
-Volumes can in version 2 only be attached on the server resource. The [above example](#scaleway_server-gt-scaleway_instance_server) shows how this works.
+Volumes can in version 2 only be attached on the server resource.
+The [above example](#scaleway_server-gt-scaleway_instance_server) shows how this works.
 
 ### Storage
 
 #### Renamed: `scaleway_bucket` -> `scaleway_object_bucket`
 
 The `scaleway_bucket` was moved to the `object` product in the `storage` product category.
 
-It's behaviour remained the same, but we also added an [`acl` attribute](../resources/object_bucket.md#acl). This attribute takes canned ACLs.
+It's behaviour remained the same, but we also added an [`acl` attribute](../resources/object_bucket.md#acl).
+This attribute takes canned ACLs.
```

Comment:
Add a section to indicate we change configuration override order. MAybe update Getting started guide also